sources/dylan/thread.dylan

```diff
@@ -30,6 +30,8 @@ define sealed class <thread> (<portable-double-container>)
 
 end class;
 
+ignore(priority);
```

Comment:
I'm not sure I understand what this does outside of a class, could you explain? Thanks! (I'd expect it to ignore an argument, kinda CL-style)